run_release.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+npm_version_params=$*
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+if [[ "$current_branch" != "master" ]]; then
+    echo 'Not in git master. Aborting release';
+    exit 1;
+fi
+
+new_tag=$(npm version $npm_version_params)
+npm run docs:publish
+git push
```

Comment:
Why the unqualified git push?

Reply:
Hmm..maybe we should commit to repo too.

Reply:
Should this be `git push origin master`?

Reply:
master + tags should only need one (`git push origin --tags master`)